sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/SubcontinentUUIDs.java

```diff
@@ -23,7 +23,7 @@
 @Getter
 public enum SubcontinentUUIDs {
   WesternEurope("Western Europe", "VMRXWX-EAGV7L-JFKP26-F3DBSBFU"),
-  CentralEurope("Central Europe", "W2FUSQ-PXGMRZ-V6ZTOE-6EPIKCSI");
+  CentralEurope("Central Europe", "SR2HAB-AY6ODA-SKBZFJ-ZMY4SJLE");
```

Comment:
why this uuid change? is it passing on test-performance as well?

Reply:
UUID was changed because previous uuid for Central Europe was un correct. This value for Central Europe is totally valid